src/components/Status/Status.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Status extends Component {
+
+  render () {
+    const { success, error } = this.props;
+
+    if (success) {
+      return (
```

Comment:
changed these values to default to undefined. removed the last conditional bracket. much more readable.